src/api.py

```
@@ -0,0 +1,154 @@
+@app.post("/process/simple")
+async def process_simple(content: str):
+    """Simple processing endpoint for backward compatibility."""
+    try:
+        response = await orchestrator.run(content)
+        return {
+            "result": response.processed_content,
+            "success": response.success,
+            "processing_time_ms": response.processing_time_ms
+        }
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)